lib/cartopy/mpl/gridliner.py

```diff
@@ -436,10 +441,6 @@ def __init__(self, axes, crs, draw_labels=False, xlocator=None,
         self._drawn = False
         self._auto_update = auto_update
 
-        # Check visibility of labels at each draw event
-        # (or once drawn, only at resize event ?)
-        self.axes.figure.canvas.mpl_connect('draw_event', self._draw_event)
```

Comment:
I don't understand why this was here.  I _think_ it's telling Matplotlib to call the method on each draw, but that was also happening by calling it within `GeoAxes.draw`.  So was this redundant or am I misunderstanding what this is for?